### DISCUSSION TITLE
[Windows][CMD] does not recognize commands

### DISCUSSION THREAD

**Discussion by Kankankan12** [Cerlancism/chatgpt-subtitle-translator#63] [General]
Clone this repository and navigate into the directory

git clone https://github.com/Cerlancism/chatgpt-subtitle-translator && cd chatgpt-subtitle-translator

Install the requirements 

npm install

I don't know how to execute these steps. i use windows 10

Give executable permission

chmod +x cli/translator.mjs

Copy .example.env to .env

cp .env.example .env

I run this command

cli/translator.mjs --help

but it doesn't recognize it
![imagen](https://github.com/user-attachments/assets/634d9ba7-ce33-4fef-b72e-492a5831236c)

**Comment by Cerlancism** [comment]
The readme assumes you have bash command line terminal. 

You are on Windows cmd, which requires some modification, here is a guide provided by ChatGPT:

https://chat.openai.com/share/7f4540c3-3b1f-4e8d-94b4-d532b05cc0b3
